psslib/driver.py

```diff
@@ -299,11 +299,13 @@ def pss_run(roots,
     if universal_newlines:
         if pattern is None:
             pattern = ''
+        openmode = 'r' if PY3 else 'rU'
```

Comment:
Would this be logically equivalent on Python 3? IIRC you need to tweak the `newline` parameter to `open` for universal newline behavior (or lack thereof)

Reply:
I think it should be good. https://docs.python.org/3/library/functions.html#open-newline-parameter states:
> `newline` controls how universal newlines mode works (it only applies to text mode). It can be None, '', '\n', '\r', and '\r\n'. It works as follows:
>
>  - When reading input from the stream, if newline is None, universal newlines mode is enabled.

As `newline` defaults to `None`.

Reply:
https://stackoverflow.com/a/56791818/1117028 has more inputs. It should be a safe change IMO.